crates/starknet-server/tests/common/mod.rs

```diff
@@ -67,6 +83,36 @@ pub mod util {
         serde_json::from_slice(&resp_body_bytes).unwrap()
     }
 
+    /// Assumes Devnet has been run with the usual account seed and returns
+    /// the signer and address of the 0th account
+    pub fn get_predeployed_account_props() -> (LocalWallet, FieldElement) {
```

Comment:
why not query the devnet for predeployed_accounts, similar functionality as the minting procedure, but making a GET request. btw its done in https://github.com/0xSpaceShard/starknet-devnet-rs/pull/117/files

Reply:
The current approach is similar to what we had in devnet-py: always (or almost always) using devnet with --seed 42, and when you need a funded account, you just take the address and the key defined in the constants. I did not rely on the endpoint for predeployed accounts because I did not think of it; I agree that it would be more stable/robust to use that approach, although somewhat slower (more complicated under the hood).